apps/meteor/app/authentication/server/startup/index.js

```diff
@@ -210,6 +212,18 @@ Accounts.onCreateUser(function (options, user = {}) {
 	}
 
 	callbacks.run('onCreateUser', options, user);
+
+	// App IPostUserCreated event hook
+	try {
+		Promise.await(Apps.triggerEvent(AppEvents.IPostUserCreated, { user, performedBy: Meteor.user() }));
+	} catch (error) {
+		if (error instanceof AppsEngineException) {
+			throw new Meteor.Error('error-app-prevented', error.message);
+		}
```

Comment:
App events that happen after the fact cannot prevent it from happening, by definition - in this case, this validation can be removed to avoid confusion. Same applies to the other triggers